build/docker/templates/intel-dlb-initcontainer.Dockerfile.in

```diff
@@ -0,0 +1,12 @@
+
+FROM ubuntu:20.04
```

Comment:
let's use the same base image than with other images

Reply:
OK :)